playbooks/quay-enterprise.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Install Quay Enterprise
+  hosts: quay_enterprise
+  vars:
+    docker_username: root
+    docker_install: false
```

Comment:
@sabre1041 the vars here should be removed, and left to the inventory to define.